yellowme/youst-kotlin
# Add collaboration guidelines

## What does this PR do?

- Add `README.md` file (First version)
- Add `CONTRIBUTING.md`
- Add `WORKFLOW.md`

## How should this be manually tested?
At Github click on the `Files changed` tab to see the markdown content. Look for the `Display the rich diff` button in order to see the final representation of the markdown file.

Please look for typos 🙏 

## What are the relevant tickets?
Closes #3
